LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2021 PickNik, Inc.
```

Comment:
This should be `PickNik Inc.` (no comma) as @nbbrooks had me do for moveit2 as that is the legal name we should use.